test/e2e/pod_ps_test.go

```diff
@@ -108,6 +108,22 @@ var _ = Describe("Podman ps", func() {
 		Expect(result).Should(Exit(0))
 	})
 
+	It("podman pod ps --filter until", func() {
+		name := mypod
```

Comment:
```suggestion
		name := "mypod"
```

Reply:
Yes, unfortunately i spotted this too late and it has to wait till afternoon :)